api/functions/getLocation/getLocation.test.js

```diff
@@ -0,0 +1,16 @@
+import { getLocation } from "./getLocation";
+
+describe("getLocation function", () => {
+	describe("when existing crs code given", () => {
+		it("returns matching long&lat value in appropriate format", async () => {
+			const expectedResponse = {
+				location: {
+					latLng: { latitude: 53.47671998, longitude: -2.228977818 },
+				},
+			};
+			expect(await getLocation("MAN")).toEqual(expectedResponse);
+		});
+	});
+});
+
+// this test is not working because test database is different than production env database
```

Comment:
Isn't this now working?

Reply:
Yes, done

Reply:
Has the comment saying it's not working been removed then? (and pushed)

Reply:
yes, it's been removed and pushed to pr